src/static/kit/scripts.js

```diff
@@ -1395,15 +1400,15 @@ function Logs(type, elem, auto_refresh_checkbox) {
 
 		var elem = content[0];
 		if (elem.scrollHeight - elem.scrollTop === elem.clientHeight &&
-			auto_refresh_checkbox.is(':checked'))
+			auto_refresh_checkbox.prop('checked'))
 		{
 			this_.try_fetching_newest();
 		}
 	}, 2000);
```

Comment:
```suggestion
	}, 2000000);
```

That is a looooot better

Reply:
Nope it is wrong